tgui/packages/tgui/interfaces/AntagInfoThief.tsx

```diff
@@ -44,10 +46,11 @@ export const AntagInfoThief = (props, context) => {
             </Section>
           </Stack.Item>
           <Stack.Item>
-            <Section textAlign="center" textColor="red" fontSize="19px">
-              Remember: You don&apos;t have a license to freely kill like
-              other antagonists.
-            </Section>
+            {!!policy && (
```

Comment:
Move this up to not render the `Stack.item`

Reply:
Moved.